advisories/github-reviewed/2023/11/GHSA-j24h-xcpc-9jw8/GHSA-j24h-xcpc-9jw8.json

```diff
@@ -47,244 +47,19 @@
               "introduced": "0"
             },
             {
-              "fixed": "4.29"
+              "fixed": "< 4.29.0"
```

Comment:
Should be just "4.29.0" afaik

Reply:
Yeah, I added the 0 based on your list though. :)

Reply:
Just checked and it does indeed have the .0:

https://central.sonatype.com/artifact/org.eclipse.platform/org.eclipse.platform/versions